addons/account/report/account_journal.py

```diff
@@ -26,75 +23,154 @@ def lines(self, target_move, journal_ids, sort_selection, data):
         if data['form'].get('target_move', 'all') == 'posted':
             move_state = ['posted']
 
         query_get_clause = self._get_query_get_clause(data)
-        params = [tuple(move_state), tuple(journal_id.ids)] + query_get_clause[2]
-        self.env.cr.execute('SELECT SUM(debit) FROM ' + query_get_clause[0] + ', account_move am '
-                        'WHERE "account_move_line".move_id=am.id AND am.state IN %s AND "account_move_line".journal_id IN %s AND ' + query_get_clause[1] + ' ',
+        params = [tuple(move_state), tuple(journal_id)] + query_get_clause[2]
+        self.env.cr.execute('SELECT "account_move_line".journal_id, SUM(debit) FROM ' + query_get_clause[0] + ', account_move am '
+                        'WHERE "account_move_line".move_id=am.id AND am.state IN %s AND "account_move_line".journal_id IN %s AND ' + query_get_clause[1] + ' GROUP BY "account_move_line".journal_id',
                         tuple(params))
-        return self.env.cr.fetchone()[0] or 0.0
+        return {journal_id: debit for journal_id, debit in self.env.cr.fetchall()}
 
     def _sum_credit(self, data, journal_id):
         move_state = ['draft', 'posted']
         if data['form'].get('target_move', 'all') == 'posted':
             move_state = ['posted']
 
         query_get_clause = self._get_query_get_clause(data)
-        params = [tuple(move_state), tuple(journal_id.ids)] + query_get_clause[2]
-        self.env.cr.execute('SELECT SUM(credit) FROM ' + query_get_clause[0] + ', account_move am '
-                        'WHERE "account_move_line".move_id=am.id AND am.state IN %s AND "account_move_line".journal_id IN %s AND ' + query_get_clause[1] + ' ',
+        params = [tuple(move_state), tuple(journal_id)] + query_get_clause[2]
+        self.env.cr.execute('SELECT "account_move_line".journal_id, SUM(credit) FROM ' + query_get_clause[0] + ', account_move am '
+                        'WHERE "account_move_line".move_id=am.id AND am.state IN %s AND "account_move_line".journal_id IN %s AND ' + query_get_clause[1] + ' GROUP BY "account_move_line".journal_id',
                         tuple(params))
-        return self.env.cr.fetchone()[0] or 0.0
+        return {journal_id: credit for journal_id, credit in self.env.cr.fetchall()}
 
     def _get_taxes(self, data, journal_id):
         move_state = ['draft', 'posted']
         if data['form'].get('target_move', 'all') == 'posted':
             move_state = ['posted']
 
         query_get_clause = self._get_query_get_clause(data)
-        params = [tuple(move_state), tuple(journal_id.ids)] + query_get_clause[2]
+        params = [tuple(move_state), tuple(journal_id)] + query_get_clause[2]
         query = """
-            SELECT rel.account_tax_id, SUM("account_move_line".balance) AS base_amount
-            FROM account_move_line_account_tax_rel rel, """ + query_get_clause[0] + """ 
+            SELECT "account_move_line".journal_id, at.id, SUM("account_move_line".balance) AS base_amount
```

Comment:
Are we well aware (functionnally speaking) that this is wrong ? (and already was)

Ideally, this should do the same as the tax report, but it's not the case. Here, we're just summing everything, so it'll be annoying for tax groups.  And I'm not even speaking about tax exigible taxes :/ This is fully inconsistent with the tax report ... If it weren't in community, we could make it so we call the report instead of doing this ... Do we really want that part of the audit report for people with only invoicing app ?